conf/application.conf

```diff
@@ -194,6 +197,8 @@ urls {
 
 accessibility-statement.service-path = "/plastic-packaging-tax"
 
+go-live-date = "2022-03-01"
```

Comment:
We use this past date locally so that we can supply a valid dates in the post-launch liability pages.